lib/datadog/open_feature/provider.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+require_relative 'ext'
+require 'open_feature/sdk'
+
+module Datadog
+  module OpenFeature
+    # Example
+    #
+    #   require 'open_feature/sdk'
+    #   require 'datadog/open_feature/provider'
+    #
+    #   Datadog.configure do |config|
+    #     config.open_feature.enabled = true
+    #   end
+    #
+    #   OpenFeature::SDK.configure do |config|
```

Comment:
it seems that for open feature to work, the customer would need to install `open_feature` gem and configure it, is this correct?